administrator/components/com_media/resources/scripts/components/browser/items/image.vue

```diff
@@ -16,7 +16,7 @@
           class="image-cropped"
           :src="getURL"
           :alt="altTag"
-          loading="lazy"
+          :loading="loading"
```

Comment:
```suggestion
          :loading="eager"
```

the loading attribute could be either `eager ` or `lazy`: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-loading